src/components/FlatmapVuer.vue

```diff
@@ -708,6 +717,15 @@ export default {
         return this.mapImp.search(term);
       return [];
     },
+    setLastClickAndCheckPopups: function(ev){
+      this.lastClick = {
+        x: ev.x,
+        y: ev.y
+      }
```

Comment:
You may need to check if the x and y here are relative to the viewport of the element or the window.
Relative to the element then everything is fine but you will have to take account of the offsets if it is the other case.